### ISSUE TITLE
Unable to open excel file generated with EPPlus 5.4.0/5.4.1

### ISSUE THREAD

**Issue by icemokka** [EPPlusSoftware/EPPlus#220]
Hi,

I'm using EPPlus to create an excel file, this is working just fine
when I try to open this excel file with EPPlus, the worksheet collection remains null.
However if I open it in excel and just click save once, then I'm able to open it properly in EPPlus too.

I have no idea how to troubleshoot or fix this :-(

Regards,
Sven Peeters

[Generated with EPPlus.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/5510564/Generated.with.EPPlus.xlsx)
[Generated with EPPlus, opened and saved in Excel.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/5510565/Generated.with.EPPlus.opened.and.saved.in.Excel.xlsx)

**Comment by JanKallman** [comment]
Hello,
EPPlus crashes due to the hyperlinks here...
```xml
<Relationship TargetMode="External" Target="#'global_optionsets'!A2" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/hyperlink" Id="rId1"/>
<Relationship TargetMode="External" Target="#'account'!A2" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/hyperlink" Id="rId2"/>
<Relationship TargetMode="External" Target="#'account_attr'!A2" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/hyperlink" Id="rId3"/>
<Relationship TargetMode="External" Target="#'opportunity'!A2" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/hyperlink" Id="rId4"/>
<Relationship TargetMode="External" Target="#'opportunity_ribbon'!A2" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/hyperlink" Id="rId5"/>
```
They are in a format not recogized correctly by EPPlus.
Can you provide the code to create these hyperlinks, if they are created with EPPlus?
I'll supply a fix for reading this format for the next version.

**Comment by icemokka** [comment]
Ji Jan,

First of all, thank you for the quick response ...

This is the code I use to create the hyperlinks :

`sheet.Cells[currentRow, 1].Value = "Global Optionsets";
                        Uri url = new Uri("#'global_optionsets'!A2", UriKind.Relative);
                        sheet.Cells[currentRow, 1].Hyperlink = url;`


Any way how I can bypass this issue for now?

Regards,
Sven Peeters

**Comment by JanKallman** [comment]
Interesting, that approach should work from the next version.
Doing it this way instead is the intended way...
`ws.Cells["K13"].Hyperlink = new ExcelHyperLink("Statistics!A1", "Statistics");`
See [Internal Hyperlink Sample](https://github.com/EPPlusSoftware/EPPlus.Sample.NetCore/blob/45d35abd3e075cfa3a524487e2e6a9eccf8882d7/20-CreateFileSystemReport/CreateAFileSystemReport.cs#L139)

**Comment by icemokka** [comment]
Hi Jan,

Your approach works better indeed ;-)

**Comment by JanKallman** [comment]
Fixed in version 5.4.2
